index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```

Comment:
Kindly, the previous reviewer suggested you check out the HTML/CSS tutorial, currently your site doesn't have a link to the CSS file, so you can't see the styles applied

Reply:
[OPTIONAL] Kindly, I suggest you always change the title of the HTML document to an appropriate one.

Reply:
@mgmediaweb I just missed that. I will make the changes and thanks for noticing it.